benches/Cargo.toml

```diff
@@ -22,70 +21,47 @@ bevy_render = { path = "../crates/bevy_render" }
 bevy_tasks = { path = "../crates/bevy_tasks" }
 bevy_utils = { path = "../crates/bevy_utils" }
 
-# make bevy_render compile on linux. x11 vs wayland does not matter here as the benches do not actually use a window
+# Other crates
+criterion = { version = "0.5.1", features = ["html_reports"] }
+glam = "0.29"
+rand = "0.8"
+rand_chacha = "0.3"
+
+# Make `bevy_render` compile on Linux with x11 windowing. x11 vs. Wayland does not matter here
+# because the benches do not actually open any windows.
 [target.'cfg(target_os = "linux")'.dev-dependencies]
 bevy_winit = { path = "../crates/bevy_winit", features = ["x11"] }
 
 [profile.release]
 opt-level = 3
 lto = true
 
-[[bench]]
-name = "change_detection"
-path = "benches/bevy_ecs/change_detection.rs"
-harness = false
-
 [[bench]]
 name = "ecs"
-path = "benches/bevy_ecs/benches.rs"
-harness = false
-
-[[bench]]
-name = "ray_mesh_intersection"
-path = "benches/bevy_picking/ray_mesh_intersection.rs"
-harness = false
-
-[[bench]]
-name = "reflect_function"
```

Comment:
I was previously running function-reflection-specific benches like:
```
cargo bench --bench reflect_function --all-features
```
Does this change mean I won't be able to run specific sets of benches anymore?

Reply:
Yes, but only temporarily! Just like unit tests, you can filter what subset of benchmarks you want to run:

```bash
# Will run `typed/function`, `typed/closure`, `typed/closure_mut`, and more.
cargo bench --bench reflect -- typed
```

My plan with #16647 is to rename all benchmarks to include their module paths, so you'll be able to run:

```bash
# Will run `reflect::function::typed/closure`, etc.
cargo bench --bench reflect -- function
```

This PR was a precursor, since I didn't want to have too many changes at once.

Reply:
I want to second this, because when I still was putting the [stress test ](https://github.com/bevyengine/bevy/pull/16591), it was very annoying to test *just* the bench while not testing everything else. Can you still run specific benchmarks (or groups of them) with this reorganization?